components/camel-undertow/src/main/java/org/apache/camel/component/undertow/UndertowConsumer.java

```diff
@@ -198,14 +200,18 @@ private void sendResponse(HttpServerExchange httpExchange, Exchange camelExchang
      * {@code connectionKey}.
      *
      * @param connectionKey an identifier of {@link WebSocketChannel} through which the {@code message} was received
+     * @param channel the {@link WebSocketChannel} through which the {@code message} was received
      * @param message the message received via the {@link WebSocketChannel}
      */
-    public void sendMessage(final String connectionKey, final Object message) {
+    public void sendMessage(final String connectionKey, WebSocketChannel channel, final Object message) {
 
         final Exchange exchange = getEndpoint().createExchange();
 
         // set header and body
         exchange.getIn().setHeader(UndertowConstants.CONNECTION_KEY, connectionKey);
+        if(channel != null) {
```

Comment:
Please run with the sourcecheck profile enabled and fix the code style.

Reply:
done